RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1759,6 +1759,12 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
             // Match inner Sg and Outer Sg
             int mIdx = alpaka::atomicAdd(acc, &matchCount, 1, alpaka::hierarchy::Threads{});
             unsigned int quintupletIndex = ranges.quintupletModuleIndices()[lowerModule1] + mIdx;
+#ifdef WARNINGS
+            printf("Quintuplet module occupancy alert! module quintuplet starting index  = %d, Pair quintuplet index = %d, next module quintuplet starting index = %d\n",
```

Comment:
Maybe the code diff is confusing me, but is this printf not wrapped inside an if statement? I think there should be an if statement here that checks if mIdx is greater than the space allocated for that module.